src/primitives/code/styles.ts

```diff
@@ -64,6 +64,7 @@ export function codeBaseStyle(): FlattenInterpolation<StyledThemeProps<Theme>> {
     }
 
     & [data-sanity-icon] {
+      display: inline;
       vertical-align: baseline;
     }
```

Comment:
Sanity UI assumes any SVG within `<Code>`, `<Text>`, `<Heading>` and `<Label>` are rendered as inline elements. So I suggest defining display for all SVGs, not only Sanity icons.

```suggestion
    & svg {
      /* Certain popular CSS libraries changes the defaults for SVG display */
      /* Make sure SVGs are rendered as inline elements */
      display: inline;
    }

    & [data-sanity-icon] {
      vertical-align: baseline;
    }
```